tests/bluetooth/iso/modules/iso_broadcast_src.c

```diff
@@ -65,8 +65,8 @@ static struct bt_iso_chan_ops iso_ops = {
 };
 
 static struct bt_iso_chan_io_qos iso_tx_qos = {
-	.sdu = sizeof(uint32_t), /* bytes */
-	.rtn = 1,
+	.sdu = 120,
```

Comment:
The default values should be Kconfigs.

Reply:
Should we set all the QoS default setting from the KConfig?